packages/manager/modules/support/src/tickets/new-ticket/translations/Messages_fr_FR.json

```
@@ -98,6 +98,7 @@
   "ovhManagerSupport_new_serviceType_saas_csp_2": "Microsoft 365 et Office 365 Reseller",
   "ovhManagerSupport_new_serviceType_horizon_view": "Horizon View",
   "ovhManagerSupport_new_serviceType_other": "Autre demande",
+  "ovhManagerSupport_new_serviceType_ai_training_job": "Ai Training",
```

Shouldn't be:

```suggestion
  "ovhManagerSupport_new_serviceType_ai_training_job": "AI Training",
```

see: https://github.com/ovh/manager/blob/master/packages/manager/modules/pci/src/projects/project/sidebar/sidebar.constant.js#L134

---

Fixed via the rebase on `master` branch.